src/wagtail_ai/views.py

```diff
@@ -55,16 +61,16 @@ def process(request):
             status=400,
         )
 
-    prompt_idx = request.POST.get("prompt")
+    prompt_id = request.POST.get("prompt")
 
     try:
-        prompt = prompts.get_prompt_by_id(int(prompt_idx))
-    except prompts.Prompt.DoesNotExist:
+        prompt = Prompt.objects.get(uuid=prompt_id)
+    except Prompt.DoesNotExist:
         return JsonResponse({"error": "Invalid prompt provided"}, status=400)
```

Comment:
**Thought:** I wonder if we need django-types installed to address those pyright errors.